static/js/disaster_dashboard_v0/page.tsx

```diff
@@ -64,7 +65,19 @@ export function Page(props: PagePropType): JSX.Element {
 
   useEffect(() => {
     // fetch date list everytime disaster type changes
-    fetchDateList(selectedDisaster)
+    const eventTypeDcids = [];
+    for (const disasterType in DISASTER_EVENT_TYPES) {
+      if (
+        selectedDisaster !== DisasterType.ALL &&
+        selectedDisaster !== disasterType
+      ) {
+        continue;
+      }
```

Comment:
I used "if condition, continue" here to prevent additional nesting to hopefully make the code easier to read. Using "if not condiiton" and then putting the following code within the if condition will create an additional level of nesting.